logging/logging_alerts/log-storage-alerts.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="log-storage-alerts"]
+include::_attributes/common-attributes.adoc[]
+= Log storage alerts
+:context: log-storage-alerts
+
+toc::[]
+
+In logging 5.7 and later versions, you can configure your LokiStack deployment to produce log-based alerts.
+
+Application, audit, and infrastructure alerts are sent to the Cluster Monitoring Operator (CMO) Alertmanager in the `openshift-monitoring` namespace by default unless you have disabled the local `Alertmanager` instance.
```

Comment:
Using the same jargon used by the monitoring docs. I'm also fine with cutting `OpenShift Container Platform monitoring stack` 
```suggestion
Application, audit, and infrastructure alerts are sent to the Alertmanager in the core OpenShift Container Platform monitoring stack under the `openshift-monitoring` namespace by default unless you have disabled the local `Alertmanager` instance.
```